app/Utils/passwordUtil.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class FreshRSS_password_Util {
```

Comment:
Could be an opportunity for that gradual namespace thing you mentioned?

Reply:
It could if we decided to go with namespaces, but we finally didn't take any decisions about this. Also as I explained, I'm not really a fan of a half done transition, that's why I'm keeping the old way :)